src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js

```diff
@@ -48,31 +45,4 @@ context('downtime notification cases for Account Security', () => {
 
     cy.injectAxeThenAxeCheck();
   });
-
-  it('should show downtime active banner, and not show the page content', () => {
-    cy.intercept('GET', '/v0/maintenance_windows', {
-      data: [
-        {
-          id: '139',
-          type: 'maintenance_windows',
-          attributes: {
-            externalService: 'emis',
-            description: 'My description',
-            startTime: beforeNow,
-            endTime,
-          },
-        },
-      ],
-    });
-    cy.visit(PROFILE_PATHS.ACCOUNT_SECURITY);
-
-    cy.injectAxeThenAxeCheck();
-
-    cy.findByText(
-      'We can’t show your account security information right now.',
-    ).should('exist');
-
-    cy.findByText('Sign in information').should('not.exist');
-    cy.findByText('Account setup').should('not.exist');
-  });
 });
```

Comment:
I think we can still test this, if we remove the emis dependency that is passed to the downtime component within the page itself and test with mvi as the externalService.

This should do the trick.

```suggestion
  it('should show downtime active banner, and not show the page content', () => {
    cy.intercept('GET', '/v0/maintenance_windows', {
      data: [
        {
          id: '139',
          type: 'maintenance_windows',
          attributes: {
            externalService: 'mvi',
            description: 'My description',
            startTime: beforeNow,
            endTime,
          },
        },
      ],
    });
    cy.visit(PROFILE_PATHS.ACCOUNT_SECURITY);

    cy.injectAxeThenAxeCheck();

    cy.findByText(
      'We can’t show your account security information right now.',
    ).should('exist');

    cy.findByText('Sign in information').should('not.exist');
    cy.findByText('Account setup').should('not.exist');
  });
});
```

Reply:
Then if we update the [Account Security component](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/components/account-security/AccountSecurity.jsx#L29)
and remove `emis` from the dependencies prop of the DowntimeNotification component that will clean it up the rest of the way for this page.

Reply:
Thank your for your input @adamwhitlock1 ! Following the above suggestions!

Reply:
Note: I will be removing the dependencies prop of DowntimeNotification in [this ticket](https://app.zenhub.com/workspaces/platform-product-team-633af4074573d06c3cda142a/issues/gh/department-of-veterans-affairs/va.gov-team/75205) next.